src/pretalx/orga/templates/orga/cfp/question_form.html

```diff
@@ -27,21 +33,23 @@ <h2>
         {% endif %}
 
         {% if action != 'create' %}
-          {% if question.active %}
-            <div class="alert alert-info col-md-9 offset-md-3">
+            {% if question.active %}
```

Comment:
Could you please revert indentation changes like this that do not add to the PR? It makes it harder to see differences and makes it harder to use `git blame` in the future.

Reply:
Hi, we tried to fix it, but we did not find a way to do it. 
We found some possible ways to recover an old version of a file on the Internet and tested them in a test repository. Some of them worked, some did not.
But we did not find a way to show the original author of the line with the git blame. 
Do you have a suggestion on how to do this?

Reply:
Don't worry about it – I'll squash/rebase some of the commits so that it will work out.